jgrapht-core/src/test/java/org/jgrapht/generate/NamedGraphGeneratorTest.java

```diff
@@ -172,6 +179,15 @@ public void testErreraGraph()
         this.validateBasics(g, 17, 45, 3, 4, 3);
     }
 
+    @Test
+    public void testFolkmanGraph()
+    {
+        Graph<Integer, DefaultEdge> g = NamedGraphGenerator.folkmanGraph();
+        this.validateBasics(g, 20, 40, 3, 4, 4);
```

Comment:
You could also use the VF2 IsomorphismInspector to count the automorphisms, but only if it doesn't run for more than a few seconds.

Reply:
After adding test for isomorphism the whole NamedGraphGeneratorTest is taking about 0.43 seconds in my laptop so I have pushed it.

Reply:
It's not mandatory, but if a graph has a well known automorphism count which can be verified quickly, then it's worth adding.  As a separate pull request, you can move your count code to a separate helper method and then reuse it in the relevant test methods.

In general, verifying well known properties (automorphism count is just one example) is useful because

(a) it verifies the correctness of the named graph implementation
(b) it provides some extra coverage for our alg code

Reply:
Okay working on the new helper method! Great idea :)